src/khoj/configure.py

```diff
@@ -328,3 +329,9 @@ def upload_telemetry():
         logger.error(f"📡 Error uploading telemetry: {e}", exc_info=True)
     else:
         state.telemetry = []
+
+
+@schedule.repeat(schedule.every(31).minutes)
```

Comment:
Why not repeat this every ~1 day (22-26 hours)?

Reply:
We'd want to flush it more frequently to give as accurate a model of usage as we can. For example, if a user made a request just after the schedule job ran to clear the cache, they would have to wait almost 48 hours for that request to clear from the DB.